portal-ui/src/screens/Console/Storage/Storage.tsx

```diff
@@ -65,7 +65,16 @@ const Storage = ({ classes, match }: IStorageProps) => {
 
   return (
     <Fragment>
-      <PageHeader label={"Storage"} />
+      <PageHeader
+        label={
+          <Fragment>
+            Storage:
```

Comment:
Please add an space next to Storage:

```suggestion
            Storage:&nbsp;
```